storm-core/src/ui/public/templates/index-page-template.html

```diff
@@ -215,6 +215,26 @@
         </span>
       </th>
       <th>
+        <span data-toggle="tooltip" data-placement="above" title="Memory capacity of a supervisor.">
+          Total Mem (MB)
+        </span>
+      </th>
+      <th>
+        <span data-toggle="tooltip" data-placement="left" title="Memory that has been allocated.">
+          Used Mem (MB)
+        </span>
+      </th>
+      <th>
+        <span data-toggle="tooltip" data-placement="above" title="CPU capacity of a supervisor. Every 100 means one core.">
+          Total CPU (%)
```

Comment:
Thanks for the comment. This is a percentage to a vCore. So every 1 equals to 1% of a core. E.g., 400 here means 4 CPU vCore. I would appreciate if we can have a more descriptive unit sign here.